images_to_samples.py

```diff
@@ -119,7 +124,15 @@ def class_proportion(target):
     return False
 
 
-def compute_classes(samples_file, data, target, metadata_idx, dict_classes):
+def compute_classes(dataset, samples_file, val_sample_file, data, target, metadata_idx, dict_classes):
+    val = False
+    if dataset == 'trn':
+        random_val = np.random.randint(1, 100)
+        if random_val > 5:
```

Comment:
So 5% is going to validation? Wouldn't it be best not to hardcode this? At least, I would document it in the README.

Reply:
You are right, best not to hardcode. Would add a val_percent param to the config file to be set by user. thanks